src/mcp/tools/crashlytics/get_sample_crash.ts

```diff
@@ -0,0 +1,50 @@
+import { z } from "zod";
+import { tool } from "../../tool.js";
+import { mcpError, toContent } from "../../util.js";
+import { getSampleCrash } from "../../../crashlytics/getSampleCrash.js";
+
+export const get_sample_crash = tool(
+  {
+    name: "get_sample_crash_for_issue",
+    description: "Gets the sample crash for an issue.",
+    inputSchema: z.object({
+      app_id: z
+        .string()
+        .describe(
+          "AppId for which the issues list should be fetched. For an Android application, read the mobilesdk_app_id value specified in the google-services.json file for the current package name. For an iOS Application, read the GOOGLE_APP_ID from GoogleService-Info.plist. If neither is available, use the `firebase_list_apps` tool to find an app_id to pass to this tool.",
+        ),
+      issue_id: z
+        .string()
+        .describe(
+          "The issue Id for which the sample crash needs to be fetched. This is the value of the field `id` in the list of issues. Defaults to the first id in the list of issues.",
```

Comment:
`Defaults to the first id in the list of issues.` should be removed from the schema. I don't know if we should be assuming that this was called after an list_top_issues call.

Reply:
It is a train of tools that backtracks. When this tool gets called, if it does not have an issue it - we are suggesting it to call the get_top_issues and then get the sample crash for that issue. Atleast, that is how I had defined it. The order of tool call cannot be guaranteed. But I'm using backtracking methods to give a sequence of tooling.

Reply:
In general pushing all of this instruction into the argument description feels funky to me. It might be the way we have to go, but it feels like it should be an anti-pattern. I don't have great reasons for it though.

Reply:
I think we should discuss about options here. Will land this change and we can circle on this offline.